<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Post2dWindowNodeVectorStreamlineGroupUnstructuredDataItem_SettingEditWidget_StartPositionEditWidget</class>
 <widget class="QWidget" name="Post2dWindowNodeVectorStreamlineGroupUnstructuredDataItem_SettingEditWidget_StartPositionEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="startPointDescLabel_2">
     <property name="text">
      <string>Specify two points. Particles start from points generated equally separated between these two points.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="pointsMouseRadioButton">
     <property name="text">
      <string>Mouse Drag</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="pointsValuesRadioButton">
     <property name="text">
      <string>Coordinate Values</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="point1Label_2">
         <property name="text">
          <string>Point 1:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="RealNumberEditWidget" name="point1XEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="point1CommaLabel_2">
         <property name="text">
          <string>,</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="RealNumberEditWidget" name="point1YEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="point2Label_2">
         <property name="text">
          <string>Point 2:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="RealNumberEditWidget" name="point2XEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="point2CommaLabel_2">
         <property name="text">
          <string>,</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="RealNumberEditWidget" name="point2YEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="numPointsLabel_2">
       <property name="text">
        <string>Number of points:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="numPointsSpinBox">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="colorLabel_2">
       <property name="text">
        <string>Color:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ColorEditWidget" name="colorWidget"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="widthLabel">
       <property name="text">
        <string>Width:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="widthSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="widthUnitLabel">
       <property name="text">
        <string>px</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorEditWidget</class>
   <extends>QLabel</extends>
   <header>guibase/widget/coloreditwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RealNumberEditWidget</class>
   <extends>QLineEdit</extends>
   <header>guibase/widget/realnumbereditwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pointsValuesRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>point1XEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pointsValuesRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>point2XEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pointsValuesRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>point1YEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pointsValuesRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>point2YEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
